src/main/kotlin/minesweeper/domain/board/strategy/DefaultRandomMineStrategy.kt

```diff
@@ -0,0 +1,17 @@
+package minesweeper.domain.board.strategy
+
+import minesweeper.domain.common.NumberSet
+import minesweeper.domain.common.PositiveInt
+import minesweeper.domain.common.until
+
+class DefaultRandomMineStrategy : MineStrategy {
```

Comment:
```suggestion
class RandomMineMaker : MineMaker {
```
전략이라는 단어를 클래스 명으로 짓는 것보다 자연스러운 이름이 좋지 않을까요?

Reply:
앗 이 부분은 MineStrategy 를 처음 구현했을때 MineIndices를 계산하는 함수 자체를 리턴하다보니 Strategy 라는 네이밍을 붙였는데요~ 리턴 타입이 변경되다보니, 이제는 Maker 가 더 적절한 이름 같습니다! 반영할게요~!